scripts/publish_video/publish_dapps.js

```diff
@@ -0,0 +1,85 @@
+require('../../page_conf')
+const { mongo } = require('../../helper');
+const { getOperations, sleep, steemPostExist, tryPublish, shouldSkip } = require('./helper');
+const moment = require('moment');
+
+const launchDate = '2023-06-15T00:00:00.000Z';
+
+(async() => {
+
+  console.log('===============================')
+
+  const videos = await mongo.Video.find({
+    status: { $in: ['publish_manual'] },
+    created: { $gte:  launchDate },
```

Comment:
Needs to be parsed as date

Reply:
Just do new Date(launchDate) and we're good to go

Reply:
on it